mobile/components/UploadedItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Image, View, Text, TouchableOpacity, StyleSheet } from 'react-native'
+
+class UploadedItem extends React.PureComponent {
+  _onPress = () => {
+    this.props.onPressItem(this.props.id)
+  }
+
+  render() {
+    const { image } = this.props
+    return (
+      <View style={styles.container}>
+        <View style={styles.imageContainer}>
+          <Image
+            style={styles.image}
+            source={{ uri: `http://localhost:3001/${image.path.replace(/^\.\//,'')}` }}
```

Comment:
Should the URI be hardcoded like this, or come from the env?

Reply:
This URI should be returned from API, it meant to add one field like `uri` on `File` for public access.